ruby/vampires.rb

```diff
@@ -0,0 +1,57 @@
+puts "How many employees will be processed?"
+employees_to_process = gets.chomp
+
+
+employees_to_process.to_i.times do
+  puts "What is your name?"
+  name = gets.chomp
+  puts "How old are you?"
+  age = gets.chomp
+  puts "What year were you born?"
+  birth_year = gets.chomp
+  puts "Do you enjoy Garlic bread? yes/no"
+  garlic = gets.chomp
+  puts "Would you like to enroll in company health insurance? yes/no"
+  insurance = gets.chomp  
+
+  allergy = nil
+  allergies = Array.new
+
+  puts "Please list any allergies. Type 'done' when finished."
+
+  until allergy == "done" || allergy == "sunshine" do     
+    allergy = gets.chomp
+    allergies << allergy
+  end
+
+
+  vampire_status = "Results inconclusive"
+  current_year = Time.new.year
+
+  age_right = current_year - birth_year.to_i == age.to_i
+  accept_garlic = garlic == "yes"
+  enroll_insurance = insurance == "yes"
+  bad_allergy = allergy == "sunshine"
+
```

Comment:
I think that you can set this up so that it does not check EVERY condition when determining the vampire results.

For instance.  If the interviewee's name is Drac or Tu then you know they are a vampire.  Is there any reason to continue the interview after that?

Try nesting your control flow in order to prevent conditions that do not need to be read from being read.